spec/bar-chart-spec.js

```diff
@@ -949,7 +949,43 @@ describe('dc.barChart', function () {
             });
         });
     });
+    describe('with elasticX and x-axis padding', function() {
+        var date = makeDate(2012, 5, 1);
+        beforeEach(function () {
 
+            var rows = [
+              {x: date, y: 4},
+            ];
+            data = crossfilter(rows);
+            dimension = data.dimension(function (d) {
+              return d.x;
+            });
+            group = dimension.group().reduceSum(function (d) {
+              return d.y;
+            });
+            chart = dc.barChart('#' + id);
+            chart.width(500)
+              .transitionDuration(0)
+              .x(d3.time.scale())
+              .elasticY(true).elasticX(true)
+              .dimension(dimension)
+              .group(group)
+            chart.render();
+        });
+        it ('should render the right xAxisMax/Min when no padding', function () {
+            expect(chart.xAxisMin()).toEqual(date);
+            expect(chart.xAxisMax()).toEqual(date);
+        })
+        it ('should render the right xAxisMax/Min when 10 day padding', function () {
+          chart.xAxisPadding(10)
+          let expectedStartDate = d3.time.day.offset(date, -10);
+          let expectedEndDate = d3.time.day.offset(date, 10);
+          expect(chart.xAxisMin()).toEqual(expectedStartDate);
+          expect(chart.xAxisMax()).toEqual(expectedEndDate);
+        });
+
+        it ('should render the right ')
```

Comment:
Ah, that might explain it. Added `expect(false).toEqual(true);` while testing earlier and all tests passed successfully. Changing to var makes tests fails as they should. I'll try to avoid ES6 syntax in the future 😄